internal/store/annotation_test.go

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package store
+
+import (
+	"github.com/mattermost/mattermost-cloud/internal/testlib"
+	"github.com/mattermost/mattermost-cloud/model"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
We usually separate the imports into groups: stdlib, third party, and first party imports.

Reply:
I guess it would make sense to enforce the imports order with `goimports` as part of CI check. I will create a ticket for that.

Reply:
That's a great idea thank you